### ISSUE TITLE
PHPUnit: There were 2 failures:

### ISSUE THREAD

**Issue by Rohrig** [rappasoft/laravel-boilerplate#1280]
# Failing Test:
1) Tests\Backend\User\ConfirmUserTest::an_admin_cannot_be_unconfirmed
   ✖ Failed asserting that null matches expected 'You can not un-confirm the super administrator.'.

2) Tests\Feature\Backend\Role\DeleteRoleTest::a_role_with_assigned_users_cant_be_deleted
   ✖ Failed asserting that null matches expected 'You can not delete the Administrator role.'.

# How to reproduce
Fresh install. Run phpunit

# What version?
6.0

**Comment by MJPTuition** [comment]
I've discovered that the RefreshDatabase trait isn't working with incremental ids. The tests pass individually, yet certain ones can fail when the whole suite is run. In this case, it's because the DeleteRoleTest is looking to delete the role with id of 1, where in my case when I dd() the role it actually had an id of 3. I've had to specify ids to get quite a few tests to pass. To fix the RefreshDatabase test, try:

`$role = Role::whereName(config('access.users.admin_role'))->first();`
`$response = $this->delete('/admin/auth/role/'.$role->id);`
